src/qt/sendcoinsdialog.cpp

```diff
@@ -35,7 +35,7 @@
 #include <QSettings>
 #include <QTextDocument>
 
-static const std::array<int, 9> confTargets = { {2, 4, 6, 12, 24, 48, 144, 504, 1008} };
+static constexpr std::array confTargets{2, 4, 6, 12, 24, 48, 144, 504, 1008};
```

Comment:
Removing a pair of `{}` causes Clang 5 to fire a false warning:
```
qt/sendcoinsdialog.cpp:38:41: warning: suggest braces around initialization of subobject [-Wmissing-braces]
static constexpr std::array confTargets{2, 4, 6, 12, 24, 48, 144, 504, 1008};
                                        ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                                        {                                  }
1 warning generated.
```

But this is not the place in the code base :)

Reply:
Indeed, this is not the only place. And generally, we don't maim our code to accommodate archaic and broken compilers.

Reply:
Not related to this PR, but we use [Clang 5](https://github.com/bitcoin/bitcoin/blob/master/ci/test/00_setup_env_native_nowallet.sh) in CI with `--enable-werror` now.
The only reason why CI passes is #18092.

Reply:
> And generally, we don't maim our code to accommodate archaic and broken compilers.

But we do accommodate our build system :)

Reply:
Yes, the build system can disable werror for clang-5, if werror for clang-5 is broken

Reply:
> Yes, the build system can disable werror for clang-5, if werror for clang-5 is broken

#20544 it its recent [reincarnation](https://github.com/bitcoin/bitcoin/pull/20544#issuecomment-738977767).